.travis.yml

```
@@ -1,18 +1,22 @@
+os: linux
+dist: bionic
 language: python
 python:
   - 2.7
   - 3.5
   - 3.6
   - 3.7
   - 3.8
+services:
+  - docker
 
 before_install:
 - pip install --upgrade pip
 - pip install flake8  # find Python syntax errors and undefined names
 - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
 install:
-- docker pull ibmcom/db2express-c
-- docker run --name db2  -p 50000:50000 -e DB2INST1_PASSWORD=password -e LICENSE=accept -d ibmcom/db2express-c db2start
+- docker pull ibmcom/db2
+- docker run --name db2  -p 50000:50000 -e DB2INST1_PASSWORD=password -e LICENSE=accept -d ibmcom/db2 db2start
```

Probably still further changes needed:
```suggestion
- docker run --name db2 -itd --privileged=true -p 50000:50000 -e LICENSE=accept -e DB2INST1_PASSWORD=password -e DBNAME=sample -v $HOME/database:/database ibmcom/db2
```

https://hub.docker.com/r/ibmcom/db2 (See "Quick Start")

Of course, the database will not be finished setting up yet:
> After the docker run command is executed, it will take a couple of minutes for the container to finish setting up.

Probably some loop is needed to check the logs for "Setup is completed", eg `docker logs db2 | grep Setup`